### ISSUE TITLE
data.ToCSV example not working

### ISSUE THREAD

**Issue by Nielius** [hairyhenderson/gomplate#804]
When I have a file `input.tmpl` with content

```
{{ $rows := (jsonArray `[["first","second"],["1","2"],["3","4"]]`) -}}
{{ data.ToCSV ";" $rows }}
```
and I run `$ gomplate -f input.tmpl`
(which is the example at the bottom of the [data functions documentation page](https://github.com/hairyhenderson/gomplate/blob/master/docs/content/functions/data.md)), I get the following error:

```
template: input.tmpl:2:7: executing "input.tmpl" at <data.ToCSV>: error calling ToCSV: Can't parse ToCSV input - must be of type [][]string
```

**Comment by Nielius** [comment]
(I'm using gomplate version 3.6.0 on macOS Catalina.)

**Comment by hairyhenderson** [comment]
Hi @Nielius, thanks for reporting this, it's definitely a bug!

Looks like somewhere along the line `jsonArray` stopped returning the right data type 🤔
Unfortunately I don't use the CSV functions very much, and the tests surrounding them are a bit lacking! 

I think the best approach to fix this will be to make `data.ToCSV` a bit more flexible - able to handle input of `[][]interface{}` as well as `[][]string`.

**Comment by hairyhenderson** [comment]
Fixed in #805

**Comment by Nielius** [comment]
Thanks!
